build.gradle

```diff
@@ -172,6 +176,34 @@ clean {
     delete sharedDir
 }
 
+dependencyCheck {
+    formats = ['SARIF', 'JSON', 'HTML']
+    failBuildOnCVSS = 5
+    suppressionFile = file('suppressions.xml')
```

Comment:
I don't think we have a `suppressions.xml` in the repository. This will cause an expectation when you try to execute the dependency check